src/GitVersionCore/Core/Abstractions/Git/ICommit.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+
+namespace GitVersion
+{
+    public interface ICommit : IEquatable<ICommit>
```

Comment:
```suggestion
    public interface ICommit : IEquatable<ICommit>, IComparable<ICommit>
```

Reply:
Agree, I will change that in a subsequent PR